benchmark/test_fused_perf.py

```diff
@@ -0,0 +1,134 @@
+import torch
+import pytest
+import flag_gems
+from .performance_utils import *
+
+
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES)
+def test_perf_gelu_and_mul(dtype):
+
+    def torch_op(x, y):
+        return torch.mul(torch.nn.functional.gelu(x), y)
```

Comment:
Looks like `torch_op` is not used in the test right? This is just for reference purpose?

Reply:
yes. fused operators can't be called by torch api. so the compared operators are torch_op and gems_op.

Reply:
I missed the baseline... It looks good.